src/data/teams/dev-tools/wgs/debugging.yml

```diff
@@ -1,5 +1,5 @@
 route: debugging
 name: Debugging working group
 description: debugging tools and foundations
-chat: "Dev-Tools Team #wg-debugging on Discord"
```

Comment:
The "Team" here and in others is in the name of the category of channels on Discord. We probably want to leave it in, but maybe add a comma or something